src/components/search/instructors/hiroko-nishimura/index.tsx

```diff
@@ -1,20 +1,146 @@
+        CTAComponent={
+          <FeaturedCourse resource={primaryCourse} location={location} />
+        }
+      />
+      <section className="flex md:flex-row flex-col max-w-screen-xl mx-auto gap-3 px-5 md:px-0 -mt-10">
+        <HorizontalResourceCard
+          resource={primaryProject}
+          location={location}
+          className="md:w-2/5"
+        />
+        <HorizontalResourceCard
+          resource={secondaryCourse}
+          location={location}
+          className="md:w-3/5"
+        />
+      </section>
+    </div>
+  )
+}
+
+export const hirokoNishimuraQuery = groq`*[_type == 'resource' && slug.current == "hiroko-nishimura-landing-page"][0]{
+  'courses': resources[slug.current == 'instructor-landing-page-featured-courses'][0]{
+    resources[]->{
+      title,
+      'description': summary,
+    	path,
+      byline,
+    	image,
+      'background': images[label == 'feature-card-background'][0].url,
+      'instructor': collaborators[]->[role == 'instructor'][0]{
+      	'name': person->.name
+    	},
+    }
+  },
+	'projects': resources[slug.current == 'instructor-landing-page-projects'][0]{
+    resources[0]{
+      title,
+      'path': url,
+      path,
+      description,
+      image,
+      byline
+    }
+  }
+}`
 
+const FeaturedCourse: React.FC<{location: string; resource: any}> = ({
+  location,
+  resource,
+}) => {
+  const {path, title, byline, description, image, background} = resource
   return (
-    <SearchInstructorEssential
-      instructor={combinedInstructor}
-      CTAComponent={<ResourceCta instructorData={instructorData} />}
-    />
+    <ExternalTrackedLink
+      eventName="clicked Hiroko Nishimura instructor page CTA"
+      params={{location}}
+      className="block md:col-span-4 rounded-md w-full h-full overflow-hidden dark:bg-gray-800 border-0 bg-white border-gray-100 shadow-sm relative text-center"
+      href={path}
+    >
+      <div className="absolute top-0 left-0 bg-gradient-to-r from-blue-700 to-blue-800 w-full h-2 z-20"></div>
+      <img
+        className="absolute h-full w-full object-cover object-left-top"
+        src={background}
+        alt=""
+      />
+      <div className="absolute inset-0 bg-gray-200 mix-blend-multiply" />
+      <div
+        className="md:-mt-5 flex items-center justify-center text-white overflow-hidden "
+        css={{
+          [bpMinMD]: {
+            minHeight: 477,
+          },
+        }}
+      >
+        <div className="relative z-10 px-5 sm:py-16 py-10 sm:text-left text-center">
+          <div className="space-y-5 mx-auto flex items-center justify-center max-w-screen-xl">
+            <div className="flex flex-col items-center justify-center sm:space-x-5 sm:space-y-0 space-y-5 gap-10 mt-10">
+              <div className="flex-shrink-0">
+                <Link href={path}>
+                  <a
+                    tabIndex={-1}
+                    onClick={() =>
+                      track('clicked jumbotron resource', {
```

Comment:
this track link is inconsistent with the `eventName` above on line 77
```suggestion
                      track('clicked Hiroko Nishimura instructor page CTA', {
```